### PR TITLE
Java client examples for CCloud.

### PR DESCRIPTION
**Pull request by arrawatia**
This PR adds examples for running java clients with Confluent cloud. It contains:

1. Producer and (new) consumer code examples.
2. Quickstart.

### CODE REVIEW

**Comment by arrawatia** [comment]
[clabot:check]

File: ccloud/java-clients/src/main/java/io/confluent/examples/clients/ProducerExample.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    long events = Long.parseLong(args[2]);
+    String topic = args[1];
+
+    // Load properties from disk.
+    Properties props = loadConfig(args[0]);
+
+    // Add additional properties.
+    props.put("acks", "all");
+    props.put("retries", 0);
```
Comment (theduderog) [inline ccloud/java-clients/src/main/java/io/confluent/examples/clients/ProducerExample.java]: Set it to 100

File: ccloud/java-clients/src/main/java/io/confluent/examples/clients/ProducerExample.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    props.put("batch.size", 16384);
+    props.put("linger.ms", 1);
```
Comment (theduderog) [inline ccloud/java-clients/src/main/java/io/confluent/examples/clients/ProducerExample.java]: This should be 50-100ms

File: ccloud/java-clients/src/main/java/io/confluent/examples/clients/ConsumerExample.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      System.exit(-1);
+    }
+
+    String topic = args[1];
+
+    // Load properties from disk.
+    Properties props = loadConfig(args[0]);
+
+    // Add additional properties.
+    props.put("group.id", "example" + new Random().nextInt());
```
Comment (theduderog) [inline ccloud/java-clients/src/main/java/io/confluent/examples/clients/ConsumerExample.java]: Add comment that this is only for example

**Pull request merged**